recipes/tensorflow-lite/all/conanfile.py

```diff
@@ -32,26 +33,18 @@ class TensorflowLiteConan(ConanFile):
         "with_mmap": True,
         "with_xnnpack": True,
     }
-    generators = "cmake", "cmake_find_package"
-    exports_sources = ["CMakeLists.txt", "patches/**"]
 
-    _cmake = None
 
-    @property
-    def _module_subfolder(self):
-        return os.path.join("lib", "cmake")
-
-    @property
-    def _module_file(self):
-        return "conan-official-{}-targets.cmake".format(self.name)
+    short_paths = True
+    generators = "cmake", "cmake_find_package", "cmake_find_package_multi"
 
     @property
     def _source_subfolder(self):
-        return "source_subfolder"
+        return "src"
```

Comment:
Why do you need this change?

Reply:
to decrease path length for windows

Reply:
You can force short_paths = True in recipe attributes

Reply:
don't you see the last commit in this PR?

Reply:
I see. But do we still need this change then? I would say, it's better to revert it